operator/hack/common.sh

```diff
@@ -114,6 +114,42 @@ function approve_install_plan() {
   retry 3 5 kubectl -n "${operator_ns}" patch installplan "${install_plan_name}" --type merge -p '{"spec":{"approved":true}}'
 }
 
+function csv_version_tag() {
+  local -r operator_ns="$1"
+  echo $(kubectl -n "${operator_ns}" get csv  -l operators.coreos.com/rhacs-operator.operators -o json | jq -r '.items[].spec.version' | sort | tail -n 1)
```

Comment:
```suggestion
  echo "$(kubectl -n "${operator_ns}" get csv -l operators.coreos.com/rhacs-operator.operators -o json | jq -r '.items[].spec.version' | sort | tail -n 1)"
```
I don't remember off the top of my head what format the `version` is in, but I would just like to point out that using a plain `sort` to sort (partially-)numeric values can go badly wrong. (Think how `sort` orders `1`, `11`, `9` vs. how `sort -n` does it).

Reply:
yeah, admittedly I used `sort` as a quick way to get the "latest" version based only on the two versions I had been testing.  I'll see if I can make this a bit less naive.